project-base/app/src/FrontendApi/Mutation/Payment/Exception/OrderCannotBePaidUserError.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\FrontendApi\Mutation\Payment\Exception;
+
+use Overblog\GraphQLBundle\Error\UserError;
+use Shopsys\FrontendApiBundle\Model\Error\UserErrorWithCodeInterface;
+
+class OrderCannotBePaidUserError extends UserError implements UserErrorWithCodeInterface
```

Comment:
I feel like this error is missing reason why not

Reply:
I'm not sure about it. That way we should either change error name, or create a new one when a new reason why order cannot be paid occurs.

Reply:
My point is: user does not know what is wrong and cannot act accordingly.

Reply:
I have separated errors